filegenerator/file_generator_test.go

```diff
@@ -0,0 +1,154 @@
+// Copyright 2016 The Cockroach Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+// implied. See the License for the specific language governing
+// permissions and limitations under the License.
+//
+// Author: William Haack (will@cockroachlabs.com)
+
+package filegenerator_test
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"math"
+	"os"
+	"os/exec"
+	"testing"
+
+	"github.com/benchviz/filegenerator"
+)
+
+func check(err error) {
+	if err != nil {
+		panic(err)
+	}
+}
+
+func TestGetGeometricMean(t *testing.T) {
+	epsilon := .00001
+	testVector := []float64{3, 9, 27}
+	result := filegenerator.GetGeometricMean(testVector)
+	if math.Abs(result-9.0) > epsilon {
+		t.Errorf("Did not properly calculate geometric mean")
+	}
+	testVector = []float64{0, 9, 27}
+	result = filegenerator.GetGeometricMean(testVector)
+	if math.Abs(result-15.58846) > epsilon {
+		t.Errorf("Did not properly calculate geometric mean")
+	}
+}
+
+func setupSampleBenchDir() {
+	check(os.Mkdir("benchSamples", 0755))
+	check(os.Mkdir("./benchSamples/01-01-2015", 0755))
+	check(os.Mkdir("./benchSamples/01-01-2015/cockroach", 0755))
+	check(os.Mkdir("./benchSamples/01-01-2015/cockroach/sql", 0755))
+	fileText := []byte("BenchmarkBank2_Cockroach-8 \t 1000 \t 1328086 ns/op \t 183965 B/op \t 2317 allocs/op")
+	f, err := os.Create("./benchSamples/01-01-2015/cockroach/sql/sql.test.stdout")
+	check(err)
+	_, _ = f.Write(fileText)
+	check(f.Sync())
+}
+
+func removeSampleBenchDir() {
+	cmd := exec.Command("rm", "-rf", "./benchSamples")
```

Comment:
`os.RemoveAll`

In general, be suspicious if you have to use `exec.Command` for basic file operations.